### PR TITLE
Fixed db script

### PR DESCRIPTION
**Pull request by michaelrtobias**

### CODE REVIEW

**Comment by michaelrtobias** [comment]
@twzkraus

File: Client/index.js [hunk 1]
```diff
@@ -2,4 +2,4 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App.jsx';
 
-ReactDOM.render(<App />, document.getElementById('app'));
+ReactDOM.render(<App />, document.getElementById('image-carousel'));
```
Comment (twzkraus) [inline Client/index.js]: should `App` be changed to `ImageCarousel`? I guess you may be doing this in the other PR

**Review by twzkraus — COMMENTED** [review]
just one question about the naming convention of App

File: Client/index.js (hunk 1 above)
Comment (twzkraus) [reply Client/index.js]: my bad--I was looking at the wrong commit. Going back to this

File: Client/App.jsx [hunk 2]
```diff
@@ -67,10 +67,10 @@ class App extends React.Component {
     }
   }
   getImagesForEndpoint() {
-    fetch('http://localhost:8080/api/products/standard-fit-hoodied-sweatshirt/baby-blue/carousel')
+    fetch('http://localhost:3001/api/products/standard-fit-hoodied-sweatshirt/baby-blue/carousel')
     .then(res => res.json())
     .then((images) => {
-
+      console.log(images[0])
```
Comment (twzkraus) [inline Client/App.jsx:73]: console log still in here

File: DB/seed.js [hunk 3]
```diff
@@ -93,14 +93,12 @@ var fakeImageDataCreator = function(numberOfEnries) {
...
+var data = realData.concat(fakeData)
 
 
-const insertImageData = function(dataArray) {
-  Image.create(dataArray)
-  .then(() => db.close())
-};
+  Image.create(data)
+  .then(() => mongoose.disconnect())
+  .catch(err => console.log(err))
```
Comment (twzkraus) [inline DB/seed.js:101]: nice, I think this will be helpful to fix the issue I was getting.

**Review by twzkraus — COMMENTED** [review]
I think this is in good shape. One console.log could be removed, otherwise ready to go

**Pull request merged**